build/cli/src/arg/wasm.rs

```diff
@@ -84,6 +106,14 @@ pub struct BuildInput {
     #[clap(long, arg_enum, enso_env())]
     pub profiling_level: Option<ProfilingLevel>,
 
+    /// Compiles Enso with given log level. If not set, defaults to [`warn`].
+    #[clap(long, arg_enum, enso_env())]
+    pub log_level: Option<LogLevel>,
```

Comment:
I'd consider prefixing this flag's name and the other one's with `wasm_` prefix. Having just a `--log-level` among tons of projects is quite generic.